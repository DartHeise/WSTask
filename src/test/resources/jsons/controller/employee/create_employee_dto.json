{
  "firstName": "Artem",
  "lastName": "Kornev",
  "description": "",
  "characteristics": ["shy", "tactful", "resourceful", "reliable"],
  "postId": "",
  "jobType": "PERMANENT",
  "contacts": {
    "phone": "+16463483212",
    "email": "Kornevenrok@gmail.com",
    "workEmail": "KornevWorker123@bk.ru"
  }
}