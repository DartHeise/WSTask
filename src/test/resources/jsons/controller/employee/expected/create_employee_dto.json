{
  "firstName": "Artem",
  "lastName": "Kornev",
  "description": "consectetur adipiscing elit",
  "characteristics": [
    "shy",
    "tactful",
    "resourceful",
    "reliable"
  ],
  "postId": "762d15a5-3bc9-43ef-ae96-02a680a557d0",
  "jobType": "PERMANENT",
  "contacts": {
    "phone": "+16463483212",
    "email": "Kornevenrok@gmail.com",
    "workEmail": "KornevWorker123@bk.ru"
  }
}